test/actor_request_factory_test.cc

```diff
@@ -75,9 +74,6 @@ TEST_F(ActorRequestFactoryShould, HaveValidParams)
     tenant_id->set_value(TENANT_ID);
 
     auto zone_offset = std::make_unique<ZoneOffset>();
-    ZoneId* zone_id = ZoneId::default_instance().New();
```

Comment:
Let's discuss why the code related to zone ID is now gone. I am not sure, since the `ActorContext` message still has those.